sonic-net/sonic-buildimage
# [202205] [Mellanox] Update SDK/FW to 4.5.4318/2010.4316 and SAI to 2205.25.1.2

#### Why I did it
Update SDK/FW to 4.5.4318/2010.4316 and SAI to 2205.25.1.2 in order to include listed below fixes.

SDK/FW

- In some cases, when an ACL has two or more rules with a similar key, modifying/removing one of the rules may cause modification/removal of one of the similar-key rules, instead of the requested rule.
- Using module SPQCELRCDFB when connected to a 3rd party switch, there may either be no link or a very long link up time (~2 minutes).
- In some case warmboot from 201911 to 202205 might result in dataplane traffic loss
- When upgrade SONiC version using warm boot from version 201911/202012 to newer version, then doing cold boot back to older version and upgrade again to newer one   warm boot might be fail.


</body>

</html>


SAI

- Added support for dynamic ordered ECMP group (SAI_NEXT_GROUP_TYPE_DYNAMIC_ORDERED_ECMP)
- "store and forward" KV was added
- Added Support for IPV6 link local debug counters


#### How I did it
Updated versions in relevant Makefiles

#### How to verify it
Build an image and verify that updated SAI/SDK/FW packages are included.
Run full sonic-mgmt test suits on various Nvidia ASICs and platforms.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
